snews.php

```diff
@@ -226,7 +226,7 @@ function readAddons() {
 readAddons();
 
 // LANGUAGE VARIABLES
-@include('lang/'.s('language');
+@include('lang/'.s('language').'.php');
```

Comment:
This is an alternative, but I think require is best.
// LANGUAGE VARIABLES
if (file_exists('lang/'.s('language').'.php')) {
	require('lang/'.s('language').'.php');
} else {
	die('Missing: lang/'.s('language').'.php');
}